File: IAL-002/Listas/4-Funções/06.py
def tamanho(n):
    # Recebe: um inteiro n (n>=0).
    # Retorna: um inteiro com a quantidade de algarismos de n.
    algarismo = 1
    while n >= 10 or n <= -10:
        n = n/10
        algarismo = algarismo+1
    return algarismo

File: ILP-010/Listas/1-Funcoes/06.c
int maior(int a[], int tam) {
  // Recebe: o endereco do vetor a e o seu tamanho tam (tam > 0).
  // Retorna: o valor do maior elemento de a.
  int i, maior = a[tam - 1];
  for (i = 0; i < tam - 1; i++) {
    if (a[i] > maior)
      maior = a[i];
  }
  return maior;
}
